custom_components/myskoda/translations/en.json

```diff
@@ -318,6 +318,15 @@
             },
             "departure_timer_3": {
                 "name": "Departure Timer 3"
+            },
+            "ac_timer_1": {
+                "name": "AirConditioning Timer 1"
+            },
+            "ac_timer_2": {
+                "name": "AirConditioning Timer 2"
+            },
+            "ac_timer_3": {
+                "name": "AirConditioning Timer 3"
```

Comment:
```suggestion
            },
            "ac_timer_1": {
                "name": "Air-conditioning Timer 1"
            },
            "ac_timer_2": {
                "name": "Air-conditioning Timer 2"
            },
            "ac_timer_3": {
                "name": "Air-conditioning Timer 3"
```

Reply:
It's either `air-conditioning` or `air conditioning`, the first seems to be most widely accepted